Prevents error:
libpng-config: --libdir option is disabled in Debian/Ubuntu

sys.platform is "android" when cross-compiling, which made find_libpng() look for
pngfix as on Windows.

--- a/setup.py
+++ b/setup.py
@@ -234,8 +234,8 @@
 
 def find_libpng():
     # Returns (found, include dir, library dir, library name)
-    if sys.platform in ("linux", "darwin", "aix"):
-        libpng_config = shutil.which("libpng-config")
+    if sys.platform in ("linux", "darwin", "aix", "android"):
+        libpng_config = "@TERMUX_PREFIX@/bin/libpng-config"
         if libpng_config is None:
             warnings.warn("libpng-config not found")
             return False, None, None, None
